content/blog/creating-a-python-aws-application-using-flask-and-redis/index.md

```diff
@@ -0,0 +1,372 @@
+---
+title: "Creating a Python AWS Application Using Flask and Redis"
+date: 2020-08-11T15:06:26-07:00
+meta_desc: A tutorial on how to create a Python AWS application using Flask, Redis, and Pulumi.
+meta_image: meta.png
+authors: ["vova-ivanov"]
+tags: ["aws", "python", "ecs", "docker"]
+---
+
+*Meet Vova Ivanov---one of the Pulumi summer interns. He'll be writing about his experiences learning Pulumi while modernizing a web app and its underlying infrastructure.*
+
+<!--more-->
+
+I've recently started developing with Pulumi, and have begun to explore its inner workings and mechanisms. I decided to construct a production-level application and to document each step that I take and my progress as I go along.
+
+This blog post recreates the existing [Typescript voting app example](https://www.pulumi.com/docs/tutorials/aws/aws-ts-voting-app/) step by step in Python with Flask as the frontend and Redis as the backend. In future blog posts, we will explore how to change the front and backends, how to upgrade the app with additional AWS services, and migrating from one cloud provider to another.
+
+The first few lines of our new `__main__.py` file indicate which libraries to import and describe a pair of configuration options used by the application
```

Comment:
where does this `__main__.py` come from - there's nothing mentioned in here about the `pulumi project` structure

Reply:
Updated the paragraph to give a quick description of the project, and what the `__main__.py` file means to it.